### ISSUE TITLE
drush queue-run stops processing

### ISSUE THREAD

**Issue by joachim-n** [drush-ops/drush#514]
drush queue-run seems to stop after a minute or so. I don't think this is due to my system's max execution time, as this:

````
php -r "print ini_get('max_execution_time');"
````

reports '0'.

Furthermore, I recently used drush to run a long database update on this system, and used `nohup drush updatedb &` and the process ran fine overnight.

Verbose output is:

````
$ drush queue-run MYQUEUE --verbose
Initialized Drupal 7.26 root directory at /var/www/yesmo                                                         [notice]
Initialized Drupal site default at sites/default                                                                 [notice]
Undefined index: time queue.drush.inc:164                                                                        [notice]
Undefined index: time queue.drush.inc:164                                                                        [notice]
Undefined index: time queue.drush.inc:164                                                                        [notice]
Undefined index: time queue.drush.inc:164                                                                        [notice]
Command dispatch complete
````

(I've filed a separate bug for the index notice.)

**Comment by davereid** [comment]
Currently the queue-run command sets an internal time limit that was originally meant to prevent queues from running long during cron. I would agree this doesn't make sense for the drush command. I'm working on removing that limit in #1434.

**Comment by davereid** [comment]
This should now be resolved with #1434, merged in master and 7.x.
